lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/ProcessCriCallbackHandler.java

```diff
@@ -271,18 +272,19 @@ public JourneyResponse getJourneyResponse(CriCallbackRequest callbackRequest)
                 criCheckingService.validatePendingVcResponse(vcResponse, clientOAuthSessionItem);
                 criStoringService.storeCriResponse(callbackRequest, clientOAuthSessionItem);
             } else {
-                vcResponse
-                        .getVerifiableCredentials()
-                        .forEach(
-                                (vc) -> {
-                                    assert criOAuthSessionItem != null;
-                                    verifiableCredentialJwtValidator.validate(
-                                            vc,
-                                            configService.getCriConfig(criOAuthSessionItem),
-                                            clientOAuthSessionItem.getUserId());
-                                });
-                criStoringService.storeCreatedVcs(
-                        vcResponse, callbackRequest, clientOAuthSessionItem);
+                for (SignedJWT vc : vcResponse.getVerifiableCredentials()) {
+                    assert criOAuthSessionItem != null;
```

Comment:
Sorry I think I missed this in the @MikeCollingwood 's review - I don't think we should be using `assert`, as it behaves rather surprisingly (i.e. it does nothing by default).

Either we should check it explicitly and throw an error, or not.

Reply:
Yep, I'll yank it. I think it's there to silence a compile time warning.